invenio_webhooks/views.py

```diff
@@ -108,11 +109,30 @@ def options(self, receiver_id=None):
         abort(405)
 
 
+class ReceiverListResource(MethodView):
```

Comment:
This should be rather CLI. I'm not sure if you want to expose all installed webhooks.